ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/BlockTransactionSelector.java

```diff
@@ -383,28 +383,12 @@ private TransactionSelectionResult handleTransactionSelected(
     if (tooLate) {
       // even if this tx passed all the checks, it is too late to include it in this block,
       // so we need to treat it as not selected
-      final var evaluationTimer = evaluationContext.getEvaluationTimer();
 
       // check if this tx took too much to evaluate, and in case remove it from the pool
-      final TransactionSelectionResult timeoutSelectionResult;
-      if (evaluationTimer.elapsed(TimeUnit.MILLISECONDS) > blockTxsSelectionMaxTime) {
-        LOG.atWarn()
-            .setMessage(
-                "Transaction {} is too late for inclusion, evaluated in {} that is over the max limit of {}ms"
-                    + ", removing it from the pool")
-            .addArgument(transaction::toTraceLog)
-            .addArgument(evaluationTimer)
-            .addArgument(blockTxsSelectionMaxTime)
-            .log();
-        timeoutSelectionResult = TX_EVALUATION_TOO_LONG;
-      } else {
-        LOG.atTrace()
-            .setMessage("Transaction {} is too late for inclusion")
-            .addArgument(transaction::toTraceLog)
-            .addArgument(evaluationTimer)
-            .log();
-        timeoutSelectionResult = BLOCK_SELECTION_TIMEOUT;
-      }
+      final TransactionSelectionResult timeoutSelectionResult =
```

Comment:
this value isn't used below? and the same thing is evaluated again when getting `actualResult`